wordpress-mobile/WordPress-iOS
# [Domain Management] Domain Details web view

Fixes #21431 and #21975

## Related PR
This PR depends on:
- https://github.com/wordpress-mobile/WordPressKit-iOS/pull/642

## Description

This PR adds the ability to view the domain details page in a web view.

| iPhone | iPad | 
| ------ | ----- |
| ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/77b4f701-645e-427c-bf69-1e69a9274121) | ![](https://github.com/wordpress-mobile/WordPress-iOS/assets/9609223/8b62f101-4b7b-4296-9435-035b6647fae5) |

## Test Instructions

1. Install Jetpack and login.
2. Enable `Domain Management` feature flag. 
3. Head to Me > All Domains.
4. Tap a domain.
5. **Expect** a navigation push to Domain Management web view.
6. Tap any button in that web page, for example "Payment Details".
7. **Expect** the new page to open in device browser.
8. Switch back to Jetpack.
9. **Expect** the Domain Management web page is still visible. 

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
None.

3. What automated tests I added (or what prevented me from doing so)
None.

## PR submission checklist:
